pkg/rootless/rootless_linux.go

```diff
@@ -110,6 +114,13 @@ func tryMappingTool(tool string, pid int, hostID int, mappings []idtools.IDMap)
 	args := []string{path, fmt.Sprintf("%d", pid)}
 	args = appendTriplet(args, 0, hostID, 1)
 	for _, i := range mappings {
+		if hostID >= i.HostID && hostID < i.HostID+i.Size {
+			what := "UID"
+			if !uid {
+				what = "GID"
+			}
+			return errors.Errorf("invalid configuration: the specified mapping %d:%d includes the host %s", i.HostID, i.Size, what)
```

Comment:
As an ignorant 1980s-era UNIX user, I would still find this message confusing and unhelpful. Please remember that subuid/subgid is a newfangled fancy thing that old people do not know about, and especially please remember that podman may be running those tools secretly in the background. I have no idea what a "mapping" is, because I the user played no part in setting it up. Could you at least include something like `/etc/sub%sid` or even just `%sid`, with a `ToLower(what)` or equivalent? It's not a huge deal, few people will ever run into this, but those who do will probably appreciate our respect for their time.

Reply:
fixed